packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.types.ts

```diff
@@ -92,6 +92,16 @@ export interface IHorizontalBarChartProps {
    * @default false
    */
   hideLabels?: boolean;
+
+  /**
+   * Callback to handle error while chart is rendering.
+   */
```

Comment:
I dont see these props being used anywhere. Why are they needed

Reply:
As discussed with Sasidhar, these props allow the users to provide their custom error handlers.

Reply:
But if they are not being used then there is no point of having them.